Commands/Public/wiki.js

```diff
@@ -0,0 +1,59 @@
+const wiki = require("wikijs").default();
+
+module.exports = async ({ Constants: { Colors } }, documents, msg, commandData) => {
+	await msg.send({
+		embed: {
+			color: Colors.INFO,
+			title: `Searching Wikipedia just for you ⌛`,
+			description: `Please stand by...`,
+		},
+	});
+	let result;
+	if (!msg.suffix) {
+		const random = await wiki.random(1);
+		result = await wiki.page(random[0]);
+	} else {
+		const search = await wiki.search(msg.suffix, 1);
+		if (!search.results.length) {
+			return msg.send({
+				embed: {
+					color: Colors.SOFT_ERR,
+					title: "What was that again? 📚🤓",
+					description: "Even Wikipedia doesn't seem to know what you're talking about.",
+					footer: {
+						text: "Check for typos or try searching for something else!",
+					},
+				},
+			});
+		}
+		result = await wiki.page(search.results[0]);
+	}
+	let description = await result.summary();
+	if (description.length < 100) {
+		// 100 is a bit short so load the full description in that case
+		description = await result.content();
+	}
+	if (description.length > 2040) {
+		description = `${description.substring(0, 2040)}...`;
+	}
+	// Sometimes wikijs crashes when attempting to grab a main image. If it works, great. If not, too bad.
+	const mainImage = await result.mainImage().catch(() => null);
+	const fields = [
+		{
+			name: "Permalink",
+			value: result.raw.fullurl,
```

Comment:
You could put this in the url property of the embed? But then we'd probably need a footer to explain that you can click on the title to access the main page

Reply:
First this.
Second, I hate the URL property. The title gets colored in ugly blue and still you don't immediately see that it's a link (because only like 0.1% of embeds are linked).
Third, on Android you'll never see it because every title is blue.